docs/tutorials/lets_write_a_test.md

```diff
@@ -1,74 +1,81 @@
 ## The Task
 
-Consider if mcollectived incorrectly spawned a new process with every puppet agent run on Ubuntu 10.04.  We need an acceptance test to check that a new process is not spawned and to ensure that this issue does not regress in new builds.
+We will write an acceptance test to check if a package (specifically HTTPD) is installed and running
 
 ## Figure Out Test Steps
 
 What needs to happen in this test:
 
-* Install PE
-* Restart mcollective twice
-* Check to see if more than one mcollective process is running
+* Install and run httpd
+  * Test that HTTPD is not installed
+  * Install HTTPD
+  * Start HTTPD service
+* Testing
+  * Test HTTPD is installed
+  * Test HTTPD is running
 
 ## Create a host configuration file
     $ beaker-hostgenerator redhat7-64ma > redhat7-64ma.yaml
 
-## Install PE
+## Install and run HTTPD
 
-We prefer to install PE once and then run a set of tests, so PE installation should not be part of the actual acceptance test.
+We prefer to install and run HTTPD once and then run a set of tests, so PE installation should not be part of the actual acceptance test. Make a file named `install.rb` and put the following code into it:
```

Comment:
Should this be "We prefer to install and run HTTPD once and then run a set of tests, so **HTTPD** installation should not be part of the actual acceptance test"?